kolibri/plugins/coach/assets/src/composables/useQuizResources.js

```diff
@@ -0,0 +1,138 @@
+import { get, set } from '@vueuse/core';
+import { computed, ref } from 'kolibri.lib.vueCompositionApi';
+import { validateObject } from 'kolibri.utils.objectSpecs';
+import { ContentNodeResource } from 'kolibri.resources';
+import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+import useFetchTree from './useFetchTree';
+import { QuizExercise } from './quizCreationSpecs.js';
+
+/**
+ * @typedef {Object} QuizResourcesConfig
+ * @property { computed <string|null|undefined> } topicId - The id of the root node to fetch the
```

Comment:
Yeah I think that @bjester mentioned during a previous code review that the proper "type" for it is called a "ComputedRef" but I ended up getting littered with errors on my JSDocs because it was not a found name so I reverted to using the functions that are imported in the file a la `ref` or `computed`, which appeased the LSP.

I'd definitely be interested to know if others' IDEs complain differently based on this.

Reply:
I think `ComputedRef` [is appropriate here](https://vuejs.org/guide/typescript/composition-api#typing-computed), and my IDE recognizes it properly. Although, it recognizes it in the sense that it does a weak match to (a) possible object(s) with the same name. I think `computed` would be incorrect because the value isn't an instance of `computed`. I can also understand your LSP not liking it if that name isn't in scope, within that file. I think having to import types starts leaning into TypeScript. I'm unsure what the best solution would be